### ISSUE TITLE
AAPT: error: resource color/material_on_background_emphasis_medium

### ISSUE THREAD

**Issue by Ratul-Bin-Tazul** [maltaisn/calcdialoglib#27]
Could not Gradle build. 

AAPT: error: resource color/material_on_background_emphasis_medium (aka com.app.smartpos:color/material_on_background_emphasis_medium) not found.

**Comment by maltaisn** [comment]
Do you have this dependency declared in your build script?
```
implementation "com.google.android.material:material:<version>"
```
If not try adding it, it might help (although in theory it shouldn't have to be necessary). If it still doesn't work after adding it, then I'll more information to be able to help you.

**Comment by anangkur** [comment]
I am facing the same issue when implementing this lib in my Dynamic Feature Module. Problem solved when I moved it to the app module. 

I implementing google material on my App module. with api. I think it's child should not implement it too.

**Comment by maltaisn** [comment]
@anangkur I managed to reproduce the issue by cloning [a sample project](https://github.com/android/app-bundle-samples/tree/master/DynamicFeatures) with dynamic features and adding my library as a dependency, like you said. 

I also found [this question on Stack Overflow](https://stackoverflow.com/questions/59755302) which seem to have the same issue, saying that all resources should exist in the base APK or something. I noticed that the sample project that I'd been using used version 1.0.0 of Google Material Components, but my library uses version 1.1.0. After changing the version in the app module everything works fine, no error. Perhaps there's no `@color/material_on_background_emphasis_medium` declared in version 1.0.0? Tell me how that works for you.
